data/json/items/gun/223.json

```diff
@@ -253,7 +272,12 @@
         "ranged_damage": 1,
         "durability": 1
     },
-    "burst": 15,
+    "modes": [
```

Comment:
You'll need evidence all these modes actually exist IRL to satisfy the aficionados. Separately from a balance POV I want to implement mods adding burst limiters so adding this functionality to lots of base guns makes this less useful.